hoverkite-protocol/src/lib.rs

```diff
@@ -276,5 +268,46 @@ mod tests {
 
             assert_eq!(round_tripped_command, command)
         }
+
+        #[test_case(SetSideLed(true))]
```

Comment:
> `+`
> Added line #L271 was not covered by tests

There are quite a lot of false positives in this coverage report thing. Mainly they are blank lines or lines with just }; on them.